tools/src/test/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommandTestUtils.java

```diff
@@ -79,34 +77,34 @@ private ConsumerGroupCommandTestUtils() {
     }
 
     static List<ClusterConfig> generator() {
-        return Stream.concat(forConsumerGroupCoordinator().stream(), forClassicGroupCoordinator().stream())
-                .collect(Collectors.toList());
+        return Stream
+            .concat(forKRaftGroupCoordinator().stream(), forZkGroupCoordinator().stream())
+            .collect(Collectors.toList());
     }
 
-    static List<ClusterConfig> forConsumerGroupCoordinator() {
+    static List<ClusterConfig> forKRaftGroupCoordinator() {
         Map<String, String> serverProperties = new HashMap<>();
         serverProperties.put(OFFSETS_TOPIC_PARTITIONS_CONFIG, "1");
         serverProperties.put(OFFSETS_TOPIC_REPLICATION_FACTOR_CONFIG, "1");
         serverProperties.put(NEW_GROUP_COORDINATOR_ENABLE_CONFIG, "true");
```

Comment:
this can be removed too.

Reply:
I was debating whether I should remove those in this PR. I can do it.